README.md

```diff
@@ -84,6 +84,26 @@ $ ./src/c2enc 1300 ../raw/hts1a.raw - | ./src/c2dec 1300 - - | play -t raw -r 80
 
 + ofdm_* are OFDM PSK HF modem command line programs (README_ofdm).
 
+## FreeDV 2020 support (building with LPCNet)
+
+1. As a pre-requisite you need libcodec2.so installed on your machine,
+as building LPCNet requires some Codec 2 functions.
+
+2. Build LPCNet:
+   ```
+   $ cd ~
+   $ git clone https://github.com/drowe67/LPCNet
+   $ cd LPCNet && mkdir build_linux && cd build_linux && cmake ../ && make
+   ```
+
+2. (re)build Codec 2 with LPCNet support:
+   ```
+   $ cd ~/codec2
+   $ rm -Rf build_linux/ && mkdir build_linux && cd build_linux
+   $ CFLAGS="-D__LPCNET__ -I/path/to/LPCNet/src" LDFLAGS=-L/path/to/LPCNet/build_linux/src cmake ..
+   $ make
+   ```
+
```

Comment:
@hobbes1069 Here's how I'm building everything atm, I've also changed the CMakeLists.txt file.  I'm sure there's a better way.

Reply:
Note it won't build without __LPCNET__ atm, as I've added ```lpcnetfreedv``` to every ```target_link_libraries```.  Hence Travis is choking.